### DISCUSSION TITLE
Improving Floppy Emulation

### DISCUSSION THREAD

**Discussion by AngelDevIndie** [dbalsom/marty#95] [Ideas]
Hi
It would be possible to support the HFE (Rev3) and IPF or SCP for disk with protection?

Thanks.

**Comment by dbalsom** [comment]
Support for more disk image formats is planned. The first formats I will probably add are ImageDisk and Teledisk formats. SCP, eventually. I am not sure about IPF or others. Availability of good documentation and/or license-compatible source code is important.

**Reply by dbalsom** [reply]
0.4.0 now has support for HFEv1 and SCP.  I've written an incomplete IPF parser, but I am not sure there are enough IPF images for PC to compile it in.
